src/models/converters/transactions/tests/summary.rs

```diff
@@ -1,419 +1,205 @@
-        };
-        let token_info = Some(TokenInfo {
-            token_type: TokenType::Erc20,
-            address: String::from("0xF9bA5210F91D0474bd1e1DcDAeC4C58E359AaD85"),
-            name: String::from("Maker"),
-            symbol: String::from("MKR"),
-            decimals: 18,
-            logo_uri: Some(String::from("https://gnosis-safe-token-logos.s3.amazonaws.com/0xF9bA5210F91D0474bd1e1DcDAeC4C58E359AaD85.png")),
+use crate::models::converters::transactions::{data_size};
+use crate::models::backend::transactions::{Transaction as TransactionDto, ModuleTransaction, EthereumTransaction, CreationTransaction};
+use crate::providers::info::*;
+use chrono::Utc;
+use crate::models::commons::Operation;
+use crate::models::service::transactions::{TransactionStatus, TransactionInfo, Custom, ID_PREFIX_ETHEREUM_TX, ID_PREFIX_CREATION_TX, ID_PREFIX_MODULE_TX, Transfer, TransferDirection, TransferInfo, EtherTransfer, Creation};
+use crate::models::service::transactions::summary::TransactionSummary;
+use crate::utils::hex_hash;
+use crate::models::backend::transfers::{EtherTransfer as EtherTransferDto, Transfer as TransferDto};
+
+#[test]
+fn data_size_calculation() {
+    assert_eq!(data_size(&None), "0");
+    assert_eq!(data_size(&Some(String::from(""))), "0");
+    assert_eq!(data_size(&Some(String::from("0x"))), "0");
+    assert_eq!(
+        data_size(&Some(String::from("0x8d80ff0a000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000f2001230b3d59858296a31053c1b8562ecf89a2f888b000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000247de7edef00000000000000000000000034cfac646f301356faa8b21e94227e3583fe3f5f001230b3d59858296a31053c1b8562ecf89a2f888b00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000024f08a0323000000000000000000000000d5d82b6addc9027b22dca772aa68d5d74cdbdf440000000000000000000000000000"))),
+        "324"
+    );
+}
+
+#[test]
+fn unknown_tx_to_summary_transaction() {
+    let unknown_tx = TransactionDto::Unknown;
+    let mut mock_info_provider = MockInfoProvider::new();
+
+    let error = unknown_tx.to_transaction_summary(&mut mock_info_provider, &String::from(""));
+
+    assert!(error.is_err());
+}
+
+#[test]
+fn module_tx_to_summary_transaction() {
+    let expected_to = String::from("0x12345789");
+    let expected_date = Utc::now();
+    let expected_date_in_millis = expected_date.timestamp_millis();
+    let module_tx = ModuleTransaction {
+        created: String::from("created"),
```

Comment:
This makes me think that we should probably define `created` as `DateTime<Utc>`. Some typing would be nice.

Reply:
These tests/ models are unrelated to this PR, but we could create an issue to remove this field, as we don't require it.